src/browser/components/SettingsPage.jsx

```diff
@@ -100,7 +115,7 @@ const SettingsPage = createReactClass({
     if (teams.length === 0) {
       this.setState({showAddTeamForm: true});
     }
-    setImmediate(this.startSaveConfig);
+    setImmediate(this.startSaveConfig, 'server');
```

Comment:
Should this string be a constant, so that it's not possible for a typo to break this functionality?